gpii/node_modules/flatMatchMaker/test/FlatMatchMakerTests.js

```diff
@@ -264,7 +261,7 @@
     gpii.flatMatchMaker.tests.disposeStrategy = function () {
         fluid.each(disposeStrategyFixtures, function (fixture) {
             var result = gpii.flatMatchMaker.disposeStrategy(fixture.leavesArg, fixture.solRecsArg);
-            var expected = $.extend(true, {}, fixture.solRecsArg, fixture.expect);
+            var expected = fluid.extend(true, {}, fixture.solRecsArg, fixture.expect);
```

Comment:
A search and replace shows that there are a lot of cases of $.extend left in the code... Is ther a reason for modifying the ones you did - or was it just a matter of which you bumped into? Perhaps it'd be worth changing all of them to fluid.extend for consistency